README.md

```diff
@@ -26,6 +26,8 @@ Anything that `new Date(input)` parses, or if that fails it will assume to be a
 relative duration starting today parsed by
 [parse-duration@^1.0.0's own micro-syntax](https://github.com/jkroso/parse-duration#available-unit-types-are).
 
+##### --config-file
+The `--config-file`(`-f`) enables to provide a path to the config file with the configurations mentioned [here](https://github.com/stefanpenner/supported/pull/36).
```

Comment:
Instead of referencing another PR, can you share an example here, then make a `CONFIGURATION.md` file or something like that and link to it from readme. You can make `CONFIGURATION.md` a placeholder file with a TODO and I can fill it in after before we cut a release.

Similar to:
https://github.com/testem/testem#configuration-file

Reply:
done